memos/umbrel-app.yml

```diff
@@ -0,0 +1,53 @@
+manifestVersion: 1
+id: memos
+category: files
+name: Memos
+version: "0.21"
+tagline: Easily capture and share your great thoughts.
```

Comment:
To stay consistent with the rest of the apps, let's remove the period at the end of the tagline.